app/models.py

```
@@ -0,0 +1,25 @@
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class ToDo(Base):
+    __tablename__ = "todos"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(
+        String(255), CheckConstraint("length(title) > 0"), index=True, nullable=False
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)